### ISSUE TITLE
Item Drops from AnvilGUI after server stop/start

### ISSUE THREAD

**Issue by Rhonim** [WesJD/AnvilGUI#96]
Currently if you stop the server while a player has an anvil menu open it will drop the item that was within the anvil when the server starts back up. Using 1.16, master-SNAPSHOT.

**Comment by WesJD** [comment]
I believe this is the case because when the server stops, the inventory close listener is no longer registered when the player closes the inventory on disconnect. Therefore, the native inventory code must drop the item. Will require looking into nms for each version to find method(s) where the inventory drops all items inside of it when closed and just override them and make them empty.

**Comment by Sean1472** [comment]
Also getting the same issue on 1.15.2 paper.

**Comment by portlek** [comment]
You can try that dependency if it's working just report  #106 here.
```xml
<dependency>
    <groupId>com.github.WesJD</groupId>
    <artifactId>AnvilGUI</artifactId>
    <version>fix-items-dropping-d0c6a1b21a-1</version>
</dependency>
```
```gradle
implementation 'com.github.WesJD:AnvilGUI:fix-items-dropping-d0c6a1b21a-1'
```

**Comment by Rhonim** [comment]
Seems to function properly with that change. Thank you!
